### ISSUE TITLE
Imported python modules cannot find JES functions

### ISSUE THREAD

**Issue by leafstorm** [gatech-csl/jes#60]
_From [cam.macd...@gmail.com](https://code.google.com/u/113022960922963935039/) on October 29, 2012 18:47:05_

What steps will reproduce the problem? 1.  Create a file that uses a JES function, like printNow
2.  Import that file from another file
3.  Run a function that calls printNow() What is the expected output? What do you see instead? I expect that printNow should work, but instead I get the error 

The error was:printNow
Name not found globally. What version of the product are you using? On what operating system? JES 4.3 on Mac OS X 10.7.4 Please provide any additional information below. In short, we'd like to be able to import user python files, but it's not working as described above.

_Original issue: http://code.google.com/p/mediacomp-jes/issues/detail?id=60_

**Comment by cayuse** [comment]
yeah same problem, is there a way to import the JES functions from within the module?

**Comment by mjguzdial** [comment]
Is the problem that you have a module that uses makePicture and getSamples and the like, and you want to know how to access those JES-specific functions?  I think that's on Page 302 in the 4th edition book: from media import *, making sure that the library is set up right.
 - Mark
